packages/SwingSet/src/vats/vat-admin/vat-vat-admin.js

```diff
@@ -418,8 +435,9 @@ export function buildRootObject(vatPowers, _vatParameters, baggage) {
       if (!pendingBundles.has(bundleID)) {
         pendingBundles.set(bundleID, makePromiseKit());
       }
-      return pendingBundles.get(bundleID).promise;
+      return pendingBundles.get(bundleID)?.promise;
```

Comment:
It’s a little weird to add a runtime check to appease the type system, when the runtime code above guarantees that the promise is defined.
```js
const bundlePromise = pendingBundles.get(bundleId);
if (bundlePromise !== undefined) {
  return bundlePromise;
}
const bundleKit = makePromiseKit()
pendingBundles.set(bundleId, bundleKit);
return bundleKit.promise;
```

Reply:
I agree that's better code but are you also asserting it's a better change to make? I opted for the minimum characters changed.

Reply:
🤷‍♂️